src/components/activity/ActivityFormPanel.svelte

```diff
@@ -41,17 +55,24 @@
   let parameterError: string | null = null;
   let startTimeField: FieldStore<string>;
   let endTime: string | null = null;
+  let editingActivityName: boolean = false;
 
   $: if ($selectedActivity) {
+    activityName = $selectedActivity.name;
     argumentsMap = $selectedActivity.arguments;
+    creationTime = $selectedActivity.created_at;
     duration = $selectedActivity.duration;
     id = $selectedActivity.id;
+    sourceSchedulingGoalId = $selectedActivity.source_scheduling_goal_id;
```

Comment:
Can we alphabetize these assignments?

Reply:
Yeah - am I missing something from ESLint / Prettier? Not seeing any warnings about these.